index.html

```diff
@@ -4683,82 +4683,134 @@ <h2>Internationalization Considerations</h2>
     </p>
 
     <section class="informative">
-      <h3>Language Declarations</h3>
+      <h3>Language and Direction</h3>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-for presentation, accessibility, and further processing to specify the language
-of the text, as follows:
+Data publishers are strongly encouraged to use the design pattern described
+in this section when expressing simple internationalized strings in
+JSON, JSON-LD, or similar data formats to ensure the broadest interoperability
+across syntaxes. The design pattern is a result of extensive collaboration
```

Comment:
I think this might be the wrong place for this. I18N has a document [String-Meta](https://www.w3.org/TR/string-meta) whose job it is to do this. I would be much happier to have you point at our document and we place whatever text we all (as a community or group thereof) agree to there. That document includes illustrations and use cases and other reference material not appropriate to VC's document and of general use by others in the Linked Data realm.

Reply:
> I would be much happier to have you point at our document and we place whatever text we all (as a community or group thereof) agree to there

Happy to do this, let's get the language at least you are happy with in the specification, and then move it to String-Meta when it's something the WG has signed off on.

Reply:
Also, we're under a time crunch, so the discussion can't take more than 1-2 weeks. I do appreciate how frustrating that is. We have WG charter pressures that we need to meet. I have no doubt we'll figure it out.

Reply:
Moved reference to STRING-META to the front matter, so people see it first in b2f3fdb952a9ca4a96411170bb5d465334cdcad3.

Reply:
I don't buy the time crunch argument. The best way out of this for VC is to avoid dragging in a lot of things you don't have to solve and let us solve it in the right place (on the Linked Data level).

Reply:
This is not just about Linked Data... it's about expression in JSON, CBOR, and a variety of other syntaxes. It is also about being able to canonicalize and digitally sign the data.

Reply:
We covered this in the call and I think it's clear now... we're trying to do the right thing and are working toward that solution with the i18n community. Resolving this conversation as it's not useful wrt. the latest PR.